cartesian_pose_haptics_controller:
    type: haptics_controllers/CartesianPoseHapticsController
    arm_id: $(arg arm_id)
    
joint_position_haptics_controller:
    type: haptics_controllers/JointPositionHapticsController
    joint_names:
    - $(arg arm_id)_joint1
    - $(arg arm_id)_joint2
    - $(arg arm_id)_joint3
    - $(arg arm_id)_joint4
    - $(arg arm_id)_joint5
    - $(arg arm_id)_joint6
    - $(arg arm_id)_joint7

